topo/node/srl/srl.go

```diff
@@ -25,7 +27,10 @@ import (
 
 const (
 	scrapliPlatformName = "nokia_srl"
-	configResetCmd      = "load factory auto-commit"
+	// srl-controller v0.6.0+ creates a named checkpoint "initial" on node startup
+	// configuration reset is therefore done by reverting to this checkpoint
+	configResetCmd = "/tools system configuration checkpoint initial revert"
+	pushCfgFile    = "/home/admin/kne-push-config"
```

Comment:
configpush functionality now undergoes the following steps:

1. config from a file is read into a string, and all double quotes are escpaed
2. the escaped config is then transferred to the srlinux node using pty transport opened with scrapligo over `kubectl exec`.
3. the transfer happens via `echo "<config lines>" > pushCfgFile` command.
4. then a candidate is opened and pushed config is sourced via `source pushCfgFile`
5. It is then committed and persisted.